### DISCUSSION TITLE
Access Point with external USB-stick (wlan1)

### DISCUSSION THREAD

**Discussion by DerRheingold** [RaspAP/raspap-webgui#1146] [Q&A]
Hi there,

t's a simple use case and I feel retarded needing to ask it here in the forum. However I couldn't find any real help for the problem via searching 😢

Use case: use a Raspi 4 with external Wifi-Antenna to set up a wifi for live-streaming (rtmp) from one device to one or more receiving devices. No internet. No bridged connection etc.

Hardware:
- Raspi 4
- Wifi: Alfa awus036achmac

Setup: 
Connected Raspi via ethernet to router, PC in same network for SSH/config. Raspi was setup according to the instructions with the onboard-wifi. Tested withouth ethernet and works as intended :) So I upgraded to the better wifi-device with better range etc.
In the running Raspi I plugged in the wifi-device via USB. It was recognized directly and I could select "wlan1" as a device. Configured it in the web-GUI and saved. Changed to the DHCP-settings, configured them from scratch as well (10.3.142.1), saved and all was working fine. Disconnected the ethernet-cable and all was well too. Cool :) 

Then shut down the Raspi, disconnected the ethernet-cable and then shit starts hitting the fan :(
- DHCP doesn't work
- Manual IP for the devices connecting doesn't help solving the problem either
- Basically I have a wifi I can't connect to
- Also can't reach the web-gui via the defined IP-address (10.3.142.1)

Unplugging the USB-cable, attaching ethernet again and then I can reach the GUI /SSH again.
Now testing the same steps with the onboard-wifi (wlan0) and all is fine. Even rebooting the raspi results in a working network etc. 

What am I missing to get the external WIFI-antenna/device to work?

Thanks for any help/advice 🤩

**Comment by billz** [comment]
Be sure the "Install a default route for this interface" option in the DHCP Server settings is **disabled**.

This is a common use case with a FAQ that addresses the steps in detail.
https://docs.raspap.com/faq/#can-i-use-wlan0-and-wlan1-rather-than-eth0-for-my-ap
